openmrs-watcher/src/test/java/org/openmrs/eip/mysql/watcher/route/WatcherRetryItemHandlerRouteTest.java

```diff
@@ -193,16 +193,16 @@ public void shouldFailIfAnEntityHasAnEventAmongFailedRetriesForTheDestination()
 	
 	@Test
 	public void shouldFailIfTheEntityDestinationIsNotAmongTheRegisteredDestinations() throws Exception {
+		SenderRetryQueueItem SenderRetryQueueItemInserted = addRetryItem("person", "1", null, MOCK_LISTENER);
```

Comment:
Since `MOCK_LISTENER` is validly used else where, I'd not change the name of the route used for this use case i.e `direct:invalid-dest` for the sake of the test being self documented?